File: Java/src/main/java/com/adaptionsoft/games/uglytrivia/PlayerNames.java
package com.adaptionsoft.games.uglytrivia;

public class PlayerNames extends Players {

    CurrentPlayer currentPlayer;

    public PlayerNames(CurrentPlayer currentPlayer) {
        this.currentPlayer = currentPlayer;
    }

    public Object get() {
        return elements.get(currentPlayer.get());
    }

    @Override
    public void println() {
        System.out.println(get() + " is the current player");
    }
}

File: Csharp/Trivia/PlayerNames.cs
using System;

namespace UglyTrivia
{
    public class PlayerNames : Players
    {
        CurrentPlayer currentPlayer;

        public PlayerNames(CurrentPlayer currentPlayer)
        {
            this.currentPlayer = currentPlayer;
        }

        public String Get()
        {
            return elements[currentPlayer.Get()];
        }

        public override void Println()
        {
            Console.WriteLine(Get() + " is the current player");
        }
    }
}
